_posts/2020-11-06-JOIN-2020.md

```diff
@@ -0,0 +1,330 @@
+
+Machine Learning models are vulnerable to attacks:
+
+- Data poisoning attacks
+  - When people can enter data or find a way to add data to a source that is being used to train a model.
+  - Verifying and cleaning data is very important to prevent this!
+- Adversarial attacks
+  - Changing a few pixels in an image might throw of the model completely
+  - Because the training dataset is so large the model becomes sensitive to small and very particular changes in the input.
+
+These attacks makes using these models for very critical things like autonomous driving tricky and potentially dangerous.
+While this problem is not fully solved yet, the field is advancing at an extremely high tempo, so these type of problems pose less and less of an issue.
+
+Machine Learning can be, and is used, for less legal & nice practices:
+
+- Spear & laser phishing
+  - Phishing attacks that use very specific data collected on the target to believe extremely believable phishing vectors.
+- Deepfakes
+  - Fake news becomes harder and harder to detect because images and even videos can be altered.
+  Faces can be swapped, even audio can be changed to match the voice of someone else.
+<div style="position: relative; width: 100%; height: 0; padding-bottom: 55%;">
+    <iframe src="https://www.youtube.com/embed/nsuAQcvafCs" width="100%" height="100%;" style="position: absolute; left: 0; top: 0; bottom: 0; right: 0;"></iframe>
+</div>
+
+- GPT3
+  - This model can write full texts, code, layouts.
+  Often the text that is generated makes more sense than anything a certain American president would say.
+<div style="position: relative; width: 100%; height: 0; padding-bottom: 55%;">
+    <iframe src="https://www.youtube.com/embed/_x9AwxfjxvE" width="100%" height="100%;" style="position: absolute; left: 0; top: 0; bottom: 0; right: 0;"></iframe>
+</div>
+
+- Recommendation engines
+  - Watch a certain type of video, get suggested more of the same but more extreme.
+  This enables a feedback loop, the type of suggested content is in your area of interest just a tad more extreme/strange/unbelievable, so you keep watching, and the next suggestions rinse and repeat the cycle.
+  This is a big issue since it has a tendency to propagate extremes.
+  What you see is not what other people see, it is curated for you by your past consumed content.
+  More than ever it is extremely important to check facts and do some digging on independent sites/new sources.
+
+## Resources
+
+- [JOIN 2020 website](https://ordina-jworks.github.io/join/){:target="_blank" rel="noopener noreferrer"}
+- [Tech Track playlist](https://www.youtube.com/playlist?list=PLgWyY-g33NlWg7rStHmTYPiN_0yJd4PxQ){:target="_blank" rel="noopener noreferrer"}
+- [Agile & Business playlist](https://www.youtube.com/playlist?list=PLgWyY-g33NlUR9F2PJ1h3bkd0cxSr4xab){:target="_blank" rel="noopener noreferrer"}
```

Comment:
There is one deleted video in the playlist?

Reply:
That's Francis's video, I'll see if I can fix this